feature-detects/proxy.js

```diff
@@ -0,0 +1,21 @@
+/*!
+{
+  "name": "Proxy Object",
+  "property": "Proxy",
+  "caniuse": "proxy",
+  "tags": ["dom"],
```

Comment:
might be an idea to put it in the dom directory if it has that tag?

Reply:
I don't think it actually has this tag so I went ahead and removed it